packages/libs/client-utils/src/marmalade/create-token.ts

```diff
@@ -88,7 +93,7 @@ const generatePolicyTransactionData = (
   if (policyConfig?.hasRoyalty) {
     data.push(addData('fungible', props.royalty.fungible));
     data.push(addData('creator', props.royalty.creator));
-    data.push(addData('creator-guard', props.royalty.creatorGuard));
+    data.push(addData('creator-guard', props.royalty.creator.keyset));
     data.push(addData('royalty-rate', props.royalty.royaltyRate.decimal));
   }
 
```

Comment:
In #L100, when checking if the config sets upgradeableURI to false, we need an enforcement that `uri_guard` is set t a failure guard.